client/packages/system/src/Item/DetailView/Tabs/ItemVariants/ItemVariantsTab.tsx

```diff
@@ -50,7 +50,7 @@ export const ItemVariantsTab = ({
         {itemVariants.length === 0 ? (
           <NothingHere
             body={t('messages.no-item-variants')}
-            onCreate={onOpen}
+            onCreate={() => onOpen()}
```

Comment:
Nice find James! I think we should actually fix this in the `NothingHere` component. 

Modal `onOpen` accepts _either_ nothing (create mode) or an existing state (for opening in edit mode.) `NothingHere` is passing the click event into the `onCreate` handler, so before this change, the click event was being passed right through and becoming the draft item variant 😱 

Then when we type in the input, the validation function was throwing because the expected fields obv weren't there.

I think pretty safe to say `onCreate` shouldn't ever want the click event, so we should update this one:

https://github.com/msupply-foundation/open-msupply/blob/a54bbe3485f8d76bba24288fa64694c83791969b/client/packages/common/src/ui/components/errors/NothingHere.tsx#L25

To `onClick={() => onCreate()}`

Reply:
Nice thanks @lache-melvin was not sure what was going on!